website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -109,9 +109,11 @@ models:
 The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html) and refreshes them for every subsequent `dbt run` that you execute. For more information, see [Refresh Materialized Views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) in the Redshift docs.
 
 Materialized views support the optional configuration `on_configuration_change` with the following values: 
-- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. The `auto_refresh` action can applied without the need to rebuild the materialized view.
+- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. 
 - `skip` &mdash; allows runs to continue while also providing a warning that the model was skipped
```

Comment:
Can we also correct this to `continue` while we're at it?

Reply:
of course